packages/hydrogen-react/src/MediaFile.tsx

```diff
@@ -52,11 +52,16 @@ export function MediaFile({
         }
       }
 
+      const {focalPoint} =
+        (data.presentation?.asJson as {focalPoint: FocalPoint} | undefined) ||
+        {};
+
       return (
         <Image
           {...passthroughProps}
           {...mediaOptions?.image}
           data={data.image}
+          focalPoint={focalPoint}
```

Comment:
`Image` objects don't contain presentation data (only [`MediaImage`](https://shopify.dev/docs/api/storefront/2024-04/objects/MediaPresentation) objects do). So instead of passing the entire presentation object through to `<Image />`, we've decided to make a `focalPoint` prop and just pass through the focal point json (e.g. `{ x: "0.5", y: "0.5"}`) - as a bonus this allows developers to more easily pass a manual focal point. Any concerns here @benjaminsehl / does this make sense to you?